bugsnag/django/middleware.py

```diff
@@ -1,12 +1,15 @@
 from __future__ import division, print_function, absolute_import
 
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
To make this backwards compatible, this call should be wrapped to ensure no import error occurs and there is a fallback base class. Something like:

```python
try:
    from django.utils.deprecation import MiddlewareMixin
except ImportError:
    MiddlewareMixin = object
```